doc/nrf/known_issues.rst

```diff
@@ -511,7 +511,7 @@ KRKNWK-7827: Application build system is not aware of the settings partition
     Set :option:`CONFIG_USE_DT_CODE_PARTITION` Kconfig option to ``y``.
     Make sure that the code partition is defined and chosen correctly (``offset`` and ``size``).
 
-.. rst-class:: v1-4-99-dev1 v1-4-2 v1-4-1 v1-4-0 v1-3-2 v1-3-1 v1-3-0
+.. rst-class:: v1-5-0 v1-4-2 v1-4-1 v1-4-0 v1-3-2 v1-3-1 v1-3-0
 
 Build configuration issues
```

Comment:
```suggestion
NCSDK-6117: Build configuration issues
```

This is the JIRA ticket, we might want to add that here.

Reply:
Added JIRA.